src/guts.rs

```diff
@@ -29,6 +29,11 @@ impl ChunkState {
         self.0.len()
     }
 
+    #[inline]
+    pub fn is_empty(&self) -> bool {
+        self.0.is_empty()
+    }
```

Comment:
This type is almost entirely internal. (Technically public, but hidden and unstable.) I'd like to avoid expanding it with convenience methods for now.